src/cli/commands/dev.ts

```diff
@@ -35,9 +35,6 @@ export async function dev(
     publicDir: staticDir ?? 'public',
     plugins: [
       devServer({
-        exclude: [
-          /.+\.(gif|jpe?g|tiff?|png|webp|bmp|woff|eot|woff2|ttf|otf|ico|txt)$/,
-        ],
```

Comment:
Was blocking devtools script from loading

Reply:
Hmm, this will break serving assets that coexist on the same level as hono routes.